best-browser-games/src/App.js

```diff
@@ -1,10 +1,12 @@
 import logo from './logo.svg';
 import './App.css';
+import CadastroForm from './components/cadastro';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
+      <CadastroForm />
+      {/* <header className="App-header">
```

Comment:
remover trecho comentado - padrão que vem no create react app
remover quando for estilizar a tela de cadastro